build/Build.cs

```diff
@@ -265,6 +276,36 @@ partial class Build : NukeBuild
                 }
             }
         });
+    Target SignPackages => _ => _
+        .DependsOn(CreateNuget)
+        .Requires(() => !SignClientSecret.IsNullOrEmpty())
+        .Requires(() => !SignClientUser.IsNullOrEmpty())
+        .Executes(() =>
+        {
+            //not sure SignClient supports .nupkg
+            //https://discoverdot.net/projects/sign-service?#user-content-supported-file-types
+            var assemblies = OutputNuget.GlobFiles("*.nupkg");            
```

Comment:
Not sure SignClient supports .nupkg according to https://discoverdot.net/projects/sign-service?#user-content-supported-file-types

Reply:
It 100% does, which is why we've been using it.

Reply:
> .nupkg via NuGetKeyVaultSignTool